scripts/alpine-setup.sh

```diff
@@ -12,6 +12,7 @@ apk add --update --no-cache \
 	make \
 	tar
 
+mkdir -p bin generate
```

Comment:
`generate` -> `generated`, but let's lave this func in the Makefile, as it will work even if someone doesn't use the install script.